Dockerfile

```diff
@@ -11,13 +11,16 @@ RUN go mod download
 # Copy the code into the container
 COPY . .
 
+# TODO: build react app?
+
 # Build the application
 RUN go build -o main
 
 # Start a new, final image to reduce size.
 FROM alpine as final
 
 # Copy the binaries and entrypoint from the builder image.
+# TODO: update for React
```

Comment:
Bumi suggested to bundle the assets directly into the executable. Maybe that can be a followup improvement unless it's easy to do

Reply:
we already do that here: https://github.com/getAlby/nostr-wallet-connect/blob/main/echo_handlers.go#L28-L32

and serving those here: https://github.com/getAlby/nostr-wallet-connect/blob/main/echo_handlers.go#L73-L75

Reply:
I removed stuff from the docker file related to this I don't think is needed